python/lsst/sims/maf/utils/bundleList2Dicts.py

```diff
@@ -0,0 +1,51 @@
+
+__all__ = ['bundleMatch', 'bundleList2Dicts']
+
+def bundleMatch(bundle1, bundle2):
+    """
+    Check if 2 metric bundle objects are compatible to be in the same
+    metricBundleGroup.
+    """
+    result = True
+    if bundle1.sqlconstraint != bundle2.sqlconstraint:
+        return False
+    for stacker1 in bundle1.stackerList:
+        for stacker2 in bundle2.stackerList:
+            if stacker1.__class__.__name__ == stacker2.__class__.__name__:
+                if stacker1 != stacker2:
+                    return False
+
```

Comment:
question: why do the stackers have to match? 
I thought we explicitly had made sure that the stackers in a metricBundleGroup were separated in the "compatible" stage if they were not the same, and that they were then run as two separate steps. 
And also that stackers could clobber previously existing columns that had the same name. 
Does this not work properly in the metricBundleGroup?  Or was there some other complication? 
The metricBundleGroups should be only sql constraint compatible, right? 
And why do you have to check the slicers? These, again, should be run separately in the metricBundleGroup (and are, now that we've fixed the problems with the slicers 'ne', as far as I can tell).